app/views/notices/shop_employee_notices/14_employee_open_enrollment_reminder.html.erb

```diff
@@ -0,0 +1,64 @@
+<%= render partial: 'notices/shared/cover', :locals => {notice: notice} %>
+<%= render partial: 'notices/shared/logo' %>
+<%= render partial: 'notices/shared/address', :locals => {notice: notice} %>
+<br><br><br><br><br>
+
+<%= render partial: 'notices/shared/date', :locals => {notice: notice} %>
+
+
+
+<table class="total_table" border="0">
+  <tbody>
+    <tr>
+      <td>Dear <%= notice.primary_fullname %>,  </td>
+    </tr>
+  </tbody>
+</table>
+
+<%= render partial: 'notices/shared/paragraph', :locals =>
+  { content: "This is a reminder that you have until #{notice.plan_year.open_enrollment_end_on} to enroll into health coverage offered by #{notice.employer_name} through the #{site_short_name}."} %>
+
+<%= render partial: 'notices/shared/paragraph', :locals =>
+  { content: "<strong>To See Plan Options and Enroll:</strong>"} %>
+
+<%= render partial: 'notices/shared/ordered_list', locals: {
+  previous: "To complete the enrollment process, please complete the following steps:",
+  listings: ["Go to <a href = '#{site_main_web_address_url}'>#{site_website_name}</a>",
+             "Log-in or create an account",
+             "If necessary, update your household information add any dependents",
+             "Select your health plan"] } %>
+
+<%= render partial: 'notices/shared/paragraph', :locals =>
+  { content: "<strong>What is Open Enrollment?</strong>"} %>
+
+<%= render partial: 'notices/shared/paragraph', :locals =>
+  { content: "Open enrollment is your annual opportunity to enroll in your employer’s health plan or make changes to your current plan selection.  Outside of this annual open enrollment period, you will only be allowed to make changes to your health plan enrollment if you experience certain life events, such as marriage, birth, or adoption, or other qualifying events."} %>
+
+<%= render partial: 'notices/shared/paragraph_with_headers', :locals =>
+    { heading: 'For Questions or Assistance:',
+      col_styles: ['margin-bottom: 0px']} %>
+<%= render partial: 'notices/shared/paragraph', :locals =>
```

Comment:
there is a partial notice_with_header partial In which we can add before header and after header may use that one.